scripts/shebangConvention.fsx

```diff
@@ -12,23 +13,10 @@ let NotInDir (dirName: string) (fileInfo: FileInfo) =
     )
 
 let invalidFiles =
-    Directory.GetFiles(".", "*.fsx", SearchOption.AllDirectories)
-    |> Seq.map(fun pathStr -> FileInfo pathStr)
-    |> Seq.filter(NotInDir "node_modules")
-    |> Seq.filter(NotInDir ".git")
-    |> Seq.filter(NotInDir "bin")
-    |> Seq.filter(NotInDir "obj")
-    |> Seq.filter(NotInDir "DummyFiles")
-    |> Seq.filter(fun fileInfo ->
-        not(FileConventions.HasCorrectShebang fileInfo)
+    Helpers.GetInvalidFiles(
+        "..",
+        "*.fsx",
+        (fun fileInfo -> not(FileConventions.HasCorrectShebang fileInfo))
     )
 
-if Seq.length invalidFiles > 0 then
-    let message =
-        "The following files don't have the correct shebang:"
-        + Environment.NewLine
-        + (invalidFiles
-           |> Seq.map(fun fileInfo -> fileInfo.FullName)
-           |> String.concat Environment.NewLine)
-
-    failwith message
+Helpers.AssertNoInvalidFiles(invalidFiles, "should have the correct shebang")
```

Comment:
@realmarv nit: concatenating words to form strings is not an internationalization-friendly technique. I say "nit" because it's not very important to fix this here, given that dev tools are very unlikely to ever translated to other languages in the future; however, it's better to do it so that you learn about this (if you don't understand what I mean with internationalization-friendly, let me know and we can discuss)

Reply:
(fixed typo above: strings->sentences*)

Reply:
Thanks for the note. What do you think I should do here? Should I pass the whole sentence `"The following files should end with EOL:"` to the function?

Reply:
yes, that's the only way to prevent future problems when doing internationalization